building/devices/stm32mp1.rst

```diff
@@ -57,6 +57,10 @@ A usual short fecth/build/load shell sequence is like the one below:
   $ make toolchains
   $ make PLATFORM=stm32mp1-157C_DK2 all
   $ dd if=../out/bin/sdcard.img of=/dev/sdX conv=fdatasync status=progress
+  $ sgdisk -e /dev/sdX
+
+Command ``sgdisk -e`` fixes the GPT redundnant data which location depends on
```

Comment:
s/redundnant/redundant

Reply:
The official terminology used in the `sgdisk` man page is: "moves backup GPT data structures to the end of the disk".

Reply:
ok, i'll change to _"Command `sgdisk -e` fixes the GPT backup data which location..."_

I'll update the commit message accordingly when squashing.